Makefile

```diff
@@ -216,7 +216,7 @@ LINT_CMD := gometalinter ./... \
 	--enable unparam \
 	--enable lll \
 	--line-length 80 \
-	--deadline 120s \
+	--deadline 240s \
```

Comment:
Probably a resource issue-- lint tests seem to take a little longer in a container executor. nbd